reana_commons/api_client.py

```diff
@@ -163,7 +163,7 @@ def submit(
         if htcondor_accounting_group:
             job_spec["htcondor_accounting_group"] = htcondor_accounting_group
 
-        if kubernetes_job_timeout:
```

Comment:
Cosmetics: Can you move `if kubenetes_job_timeout` clause higher, so that all `if kubernetes_*` would be presented  together, and condors together?